_data/simple-icons.json

```diff
@@ -9941,7 +9947,7 @@
         {
             "title": "WebAuthn",
             "hex": "3423A6",
-            "source": "https://github.com/webauthn-open-source/webauthn-logos",
+            "source": "https://github.com/webauthn-open-source/webauthn-logos/blob/master/final-webauthn-logo-logo-black.svg",
```

Comment:
Should be https://github.com/webauthn-open-source/webauthn-logos/blob/b21be672811eb4a5caadaba41044970cae299a55/final-webauthn-logo-logo-black.svg